### DISCUSSION TITLE
请问Docker Run 部署的如何升级~？

### DISCUSSION THREAD

**Discussion by MaoVlog** [usememos/memos#732] [Q&A]
我之前是在Linux 直接Docker Run 
`docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest  `
命令部署的。
现在如何升级？
求大神指教~谢谢~

**Comment by Zeng1998** [comment]
https://github.com/usememos/memos/issues/624#issuecomment-1329102570

**Comment by loudtell** [comment]
运行这个，可以自动升级所有的Docker容器。

docker run --rm --name watchtower -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --cleanup --run-once

**Reply by MaoVlog** [reply]
好的~谢谢!
